package.json

```diff
@@ -113,9 +113,9 @@
     "appname": "connector"
   },
   "routes": {
-    "connector": "/",
+    "connector": "/*",
     "connectSystemsModal": "/connect-systems",
-    "logModal": "/log",
+    "logModal": "log",
     "activationKeys": "/activation-keys",
```

Comment:
I think these paths here should still have all the leading `/` removed.

This will for example allow the path in `/Components/ActivationKey/ActivationKey.js` to stay the same and the import of `useNavigate` can be changed to import the `useInsightsNavigate` hook.

It will also make the routes consistent and there is no question or need for investigation later on, why some routes do and some routes do not have a `/` in front.